### PR TITLE
Ci dev

### PR DESCRIPTION
**Pull request by tomtrafford**
Adds CI to build the webcontrol zpkg and the docs.

### CODE REVIEW

**Comment by tomtrafford** [comment]
I'm not certain if this is the correct way to build the docs. Does this need a conf.py like the other repos?

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      - uses: rishabhgupta/split-by@v1
+        id: split
+        with:
+          string: ${{ github.ref }}
+          split-by: /
+
+      # Publish docs
+      - name: Move to versioned directory
+        # e.g. master or 0.1.2
+        run: mv PandABlocks-webcontrol/build/templates "PandABlocks-webcontrol/.github/pages/${{ steps.split.outputs._2 }}"
```
Comment (thomascobb) [inline .github/workflows/docs.yml]: Found a better way to do this:
https://github.com/dls-controls/scanspec/blob/master/.github/workflows/docs.yml#L41-L43

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
+PandABlocks-webcontrol
+======================
+
+|code_ci| |license|
```
Comment (thomascobb) [inline README.rst]: Need docs_ci badge here

File: .github/workflows/docs.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        uses: actions/checkout@v2
+        with:
+          repository: dls-controls/annotypes
+          path: annotypes
+
+      - name: Checkout pymalcolm
+        uses: actions/checkout@v2
+        with:
+          repository: dls-controls/pymalcolm
+          path: pymalcolm
```
Comment (thomascobb) [inline .github/workflows/docs.yml]: You might not need these two, I can't remember what make docs looks at...

**Review by thomascobb — COMMENTED** [review]
This is the right approach, we basically take malcolmjs docs and hack it about

**Pull request merged**